TYPO3.Neos/Documentation/Appendixes/ReleaseNotes/220.rst

```diff
@@ -0,0 +1,114 @@
+=====
+2.2.0
+=====
+
+
+User experience improvements
+============================
+
+- The styling of the user interface has been improved in many places (`checkboxes <https://github.com/neos/neos-development-collection/pull/457>`_, `dropdowns <https://github.com/neos/neos-development-collection/pull/459>`_, radio buttons, `active inspector tab <https://github.com/neos/neos-development-collection/pull/456>`_, `help message icon <https://github.com/neos/neos-development-collection/pull/453>`_, `error notifications <https://github.com/neos/neos-development-collection/pull/424>`_, …)
+- `Inspector groups can now be assigned icons <https://github.com/neos/neos-development-collection/pull/455>`_ to help identify them more easily.
+- The structure tree has been improved (`configurable loading depth <https://github.com/neos/neos-development-collection/pull/451>`_, `selectable nodes <https://github.com/neos/neos-development-collection/pull/443>`_, `node type changes <https://github.com/neos/neos-development-collection/pull/442>`_, …).
+- All `dialogs are centered vertically <https://github.com/neos/neos-development-collection/pull/434>`_ now.
+- The Workspaces module has been improved with better date/time handling, styling fixes, …
+- Insert panel groups `are now collapsible <https://github.com/neos/neos-development-collection/pull/418>`_.
+- `Sortable options in selector editors <https://github.com/neos/neos-development-collection/pull/338>`_  (select, references, asset list).
+- Editors can now `switch between sites without logging in again <https://github.com/neos/neos-development-collection/pull/356>`_.
+- Preview uses the `target workspace instead of live <https://github.com/neos/neos-development-collection/pull/399>`_ if applicable.
```

Comment:
would add
- [ ] error notifications have been improved (no more tall orange notifications) (https://github.com/neos/neos-development-collection/pull/424)
- [ ] insert new/paste position selection is more intuitive (opens automatically, only one click needed) – additionally the selected position is shown in the insert new panel (https://github.com/neos/neos-development-collection/pull/426 / https://github.com/neos/neos-development-collection/pull/320)
- [ ] lots of new tooltips and a new way of displaying them (https://github.com/neos/neos-development-collection/pull/320)